Update README.md (#1)
ArthurNRL/python-coverage-comment-action: README.md

A previous version of this action did things with the wiki. This is not the case
anymore.

> ## .coverage file generated on a non-unix file system

Reviewer: ```suggestion
## .coverage file generated on a Windows file system
```

> If your project needs to be built and tested on a non-unix os adding 

Reviewer: `````suggestion
If your project's coverage was built on Windows, you may get an error like:
```
CoverageWarning: Couldn't parse 'yourproject\__init__.py': No source for code: 'yourproject\__init__.py'. (couldnt-parse)
```
This is likely due to coverage being confused with the coverage being computed with `\` but read with `/`. You can most probably fix it with the following in your [coverage configuration](https://coverage.readthedocs.io/en/latest/config.html):
`````

```
[paths]
source =
    */project/module
    *\project\module
```

> 
> to .coveragerc will help the action to find the covered files.

Reviewer: ```suggestion
```
